File: nodeJS/chatgpt_API.js
import { Configuration, OpenAIApi } from "openai";
const configuration = new Configuration({
    organization: "org-mKiOdj05I0xVUxoQeuGVMiVl",
    apiKey:"OPENAI_API_KEY",
});
const openai = new OpenAIApi(configuration);
const completion = await openai.createChatCompletion({
    model: "gpt-3.5-turbo",
    messages: [{role: "user", content: "Write a python hello world"}],
    max_tokens:70,
  });
console.log(completion.data.choices[0].message);

File: python/chatgpt_API.py
import os
import openai
openai.api_key = "OPENAI_API_KEY"
completion = openai.ChatCompletion.create(
  model="gpt-3.5-turbo",
  messages=[
    {"role": "user", "content": "Write a JavaScript hello world"}
  ],
  max_tokens = 70,
)

print(completion.choices[0].message)
